ga4gh/datamodel/rna_quantification.py

```diff
@@ -0,0 +1,390 @@
+        self._analysisId = record["rna_quantification_id"]
+        self._name = record["feature_group_id"]
+
+    def toProtocolElement(self):
+        protocolElement = protocol.FeatureGroup()
+        protocolElement.id = self._id
+        protocolElement.analysisId = self._analysisId
+        protocolElement.name = self._name
+        return protocolElement
+
+
+class AbstractRNAQuantification(datamodel.DatamodelObject):
+    """
+    An abstract base class of a RNA quantification
+    """
+    compoundIdClass = datamodel.RnaQuantificationCompoundId
+
+    def __init__(self, parentContainer, localId):
+        super(AbstractRNAQuantification, self).__init__(parentContainer,
+                                                        localId)
+        self._annotationIds = []
+        self._description = ""
+        self._name = localId
+        self._readGroupId = ""
+
+    def toProtocolElement(self):
+        """
+        Converts this rnaQuant into its GA4GH protocol equivalent.
+        """
+        protocolElement = protocol.RnaQuantification()
+        protocolElement.annotationIds = self._annotationIds
+        protocolElement.description = self._description
+        protocolElement.id = self.getId()
+        protocolElement.name = self._name
+        protocolElement.readGroupId = self._readGroupId
+        return protocolElement
+
+    def addRnaQuantMetadata(self, fields):
+        """
+        data elements are:
+        Id, annotations, description, name, readGroupId
+        where annotations is a comma separated list
+        """
+        self._annotationIds = fields["annotation_ids"].split(',')
+        self._description = fields["description"]
+        self._name = fields["name"]
+        self._readGroupId = fields["read_group_id"]
+
+
+class RNASeqResult(AbstractRNAQuantification):
+    """
+    Class representing a single RnaQuantification in the GA4GH data model.
+    """
+
+    def __init__(self, parentContainer, localId, rnaQuantDataPath,
+                 dataRepository):
+        super(RNASeqResult, self).__init__(parentContainer, localId)
+        self._dbFilePath = rnaQuantDataPath  # the full path of the db file
+        self._dataRepository = dataRepository
+        self._db = SqliteRNABackend(self._dbFilePath)
+        self.getRnaQuantMetadata()
+
+    def getRnaQuantMetadata(self):
+        """
+        input is tab file with no header.  Columns are:
+        Id, annotations, description, name, readGroupId
+        where annotation is a comma separated list
+        """
+        rnaQuantId = self.getLocalId()
+        with self._db as dataSource:
+            rnaQuantReturned = dataSource.getRnaQuantificationById(
+                rnaQuantId)
+        if rnaQuantReturned is not None:
+            self.addRnaQuantMetadata(rnaQuantReturned)
+        else:
+            raise exceptions.RnaQuantificationNotFoundException(rnaQuantId)
+
+    def getExpressionLevels(self, rnaQuantID, pageToken=0, pageSize=None,
```

Comment:
Probably better to use hanging indent in situations like this where future code changes will break the alignment, eg.
```python
def getExpressionLevels(
    self, rnaQuantID, pageToken=0, pageSize=None,  expressionId=None, 
    featureGroupId=None,threshold=0.0):
```